SUM function no longer determines values in the sum( 'Column name'/number/number/number) configs with the first row being empty

alsundukov opened the pull request.

Commit:
TBLFILTER-3272
TT: SUM function no longer determines values in the sum( 'Column name'/number/number/number) configs with the first row being empty

dkostochko commented on src/423groupby.js:
```diff
@@ -282,7 +282,7 @@ yy.Select.prototype.compileGroup = function (query) {
 					}
 					if (col.aggregatorid === 'SUM') {
-						return `${pre}if((y=${colexp})!=null)g['${colas}']+=y;${post}`;
+						return `${pre}if((y=${colexp})!=null){ if(g['${colas}']==null)g['${colas}']=y; else g['${colas}']+=y;} ${post}`;
```
Может вынести if  в переменную вычисленную как тернарный оператор?

Commit:
TBLFILTER-3272
TT: SUM function no longer determines values in the sum( 'Column name'/number/number/number) configs with the first row being empty

Used a ternary operator.
